v0/destinations/mailchimp/transform.js

```diff
@@ -100,6 +100,19 @@ async function responseBuilderSimple(payload, message, mailChimpConfig) {
   const basicAuth = Buffer.from(`apiKey:${mailChimpConfig.apiKey}`).toString(
     "base64"
   );
+  const validStatus = [
```

Comment:
Can we move `validStatus` to config.js  and import it and use it here, make the code clean here

Reply:
Did that.